stl/inc/filesystem

```diff
@@ -3599,8 +3607,17 @@ namespace filesystem {
     inline bool remove(const path& _Target, error_code& _Ec) noexcept {
         // remove file _Target (even if it is a directory); returns whether the file was removed
         // note !exists(_Target) is not an error, and merely returns false
-        const auto _Result = __std_fs_remove(_Target.c_str());
-        _Ec                = _Make_ec(_Result._Error);
+        auto _Result = __std_fs_remove(_Target.c_str());
+        if (_Result._Error == __std_win_error::_Access_denied) {
+            // Try to remove the ReadOnly Attribute and try again

```

Comment:
Should this phrase be capitalized? I would expect:
```suggestion
            // Try to remove the read-only attribute and try again
```

Reply:
Will change it